s3/pkg/meta/db/drivers/tidb/bucket.go

```diff
@@ -184,6 +192,14 @@ func (t *TidbClient) GetBuckets(ctx context.Context) (buckets []*Bucket, err err
 			tmp.Versioning.Status = versionOpts.Status
 		}
 
+		// get SSE info for this bucket
```

Comment:
Is it necessary to return sse configuration when list buckets?

Reply:
hmm...this is used to list the buckets, SSE is one attribute of a bucket. So if we have 100 buckets, one call will return 100 bucket info, including SSE, instead of 200 calls, for example, to list the SSE column on dashboard, for each bucket